includes/class-wc-install.php

```diff
@@ -1078,8 +1078,9 @@ public static function plugin_row_meta( $links, $file ) {
 	 * @param string $key Plugin relative path. Example: woocommerce/woocommerce.php.
 	 */
 	private static function associate_plugin_file( $plugins, $key ) {
-		$path                = explode( '/', $key );
-		$plugins[ $path[1] ] = $key;
+		$path                 = explode( '/', $key );
+		$filename             = array_slice( $path, -1 )[0];
```

Comment:
I dont believe [this syntax is valid on old PHP versions](https://travis-ci.org/woocommerce/woocommerce/jobs/375884753). If I'm reading this correctly, can you just use [`end`](http://php.net/manual/en/function.end.php)?

Reply:
Ah, right — fixed in https://github.com/woocommerce/woocommerce/pull/19987/commits/d7dc2d45a4bc1c1cabed53054a3a20601736e182